changelog.yaml

```diff
@@ -52,6 +52,16 @@
   date: TBD
   changes:
 
+    - type: enhancement
+      impact: patch
```

Comment:
As Jenkins core and all plugins were updated then I'd expect this change to be a minor version not a patch version

Reply:
Please also update `charts/steward/Chart.yaml` file accordingly

Reply:
Valid point. Done with b53b3c237893e770e9c4b42395e6caa9e076ead3.